### ISSUE TITLE
Dropdown is inaccessible in Windows High Contrast Mode

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#6880]
## Environment

> Operating system

Windows 10 latest

> Browser

Edge, Firefox

> Assistive technology used to verify

Windows High Contrast Mode

## Detailed description

The borders of the listbox are weirdly gone in Firefox when compared to Edge. This is behavior I've seen in a few other components.

Firefox
![image](https://user-images.githubusercontent.com/40970507/93833990-f8786380-fc3f-11ea-9a9b-bf224a9aa676.png)
Edge
![image](https://user-images.githubusercontent.com/40970507/93833997-fd3d1780-fc3f-11ea-8c06-9304d8e97380.png)

There is also no focus indicator or iconography

**Comment by tw15egan** [comment]
![Screenshot 2021-03-04 180104](https://user-images.githubusercontent.com/11928039/110056408-9dc95480-7d13-11eb-98a3-4f4e6e83cce9.png)

Seems to be resolved

Found a separate issue that is affecting all Dropdowns will fix in a separate issue (#7989)
![Screenshot 2021-03-04 180257](https://user-images.githubusercontent.com/11928039/110056539-e08b2c80-7d13-11eb-900e-57f6da5d1845.png)
